show/main.py

```diff
@@ -1469,6 +1469,20 @@ def ssdhealth(device, verbose, vendor):
     options += " -e" if vendor else ""
     run_command(cmd + options, display_cmd=verbose)
 
+# 'fanstatus' subcommand ("show platform fanstatus")
+@platform.command()
+def fanstatus():
```

Comment:
I understand we currently have `show platform psustatus`, but I wonder if we should shorten this command to simply, `show platform fans` or `show platform fan`, and in the future, shorten the other command to `show platform psu`.